src/plugins/intel_cpu/src/nodes/eltwise.cpp

```diff
@@ -1885,27 +1934,6 @@ void Eltwise::initSupportedPrimitiveDescriptors() {
     currentInBlkDims.resize(inputNum);
 }
 
-std::vector<VectorDims> Eltwise::shapeInfer() const {
-    ov::PartialShape outShape = getParentEdgesAtPort(0)[0]->getMemory().GetShape().toPartialShape();
-    for (size_t i = 1; i < getParentEdges().size(); i++) {
-        ov::PartialShape::broadcast_merge_into(outShape, getParentEdgesAtPort(i)[0]->getMemory().GetShape().toPartialShape(),
-        ov::op::AutoBroadcastType::NUMPY);
-    }
-
-    if (outShape.is_dynamic()) {
```

Comment:
This check is missed now as far as I can see. Is it expected?

Reply:
My understanding is that this condition was added explicitly to throw meaningful error in case shape infer still produced dynamic output shape. This is quite often case that somebody tries to pass invalid/unsupported shapes on dynamic inputs. The message much better helps to understand the source of the problem than smt like "Cannot get static shape" which we throw by default.

Reply:
Shape inference call now wrapped in try catch block to provide additional information (node type and name) to the exception message.